mesh/mesh.hpp

```diff
@@ -722,6 +727,30 @@ class Mesh
       double sx = 1.0, double sy = 1.0, double sz = 1.0,
       bool sfc_ordering = true);
 
+   /** Creates a Tet mesh by splitting each hex in inline-hex mesh into
+       24 (type = 2) tets. Each face of the hex is split into 4 triangle
+       (face edges are connected to a face-centered point), and the triangles
+       are connected to a hex-centered point.
+   */
+   static Mesh MakeCartesian3DWith24TetsPerHex(int nx, int ny, int nz,
+                                               double sx = 1.0, double sy = 1.0,
+                                               double sz = 1.0);
+
+   /** Creates a triangle mesh by splitting each quad in inline-quad mesh
+       into 4 triangles. Each corner is connected to the element center to
+       form 4 triangles.
+   */
+   static Mesh MakeCartesian2DWith4TrisPerQuad(int nx, int ny,
+                                               double sx, double sy);
+
+   /** Creates a quad mesh by splitting each quad in inline-quad mesh
+       into 5 Quads. Each Quad is Projected inwards and connected to the
```

Comment:
projected (lower case "p")

Reply:
I would prefer whole words are used here and in other comments, i.e, quadrilateral, hexahedron, etc.